data/human/human missions.txt

```diff
@@ -3996,3 +3996,91 @@ mission "Fumbling Mugger"
+			`	Suddenly, proximity sensors go off on the starboard side and a camera shows something - or someone - bounce off your shield and tumble down the mountain.`
+			choice
+				`	(Announce that I'm leaving).`
+					goto leave
+				`	(Stay and watch the Dragonriders).`
+					goto watch
+				`	(Take off and leave everybody else behind).`
+					goto betrayal
+			label watch
+			`	Checking your camera feeds again, you see that object rolling down the hill is indeed a body. On another monitor, one of the Dragonriders is keeping just above a dragon, attempting to lower himself down. The dragon rolls in midair, its wing catching the man and sending him flying out of control. Fortunately, a leafy tree branch prevents any further physical harm. Meanwhile, a second dragon is pursuing another one of the Dragonriders.`
+			choice
+				`	(Get everyone out of there).`
+					goto leave
+				`	(Take off and leave everybody else behind).`
+			label betrayal
+			`	The ship takes off with a roar at your touch. Between the cries of the dragons, the sound of the engines, and the air rushing past, you can barely hear the fear and betrayal in the voices of the men you've left behind.`
+				fail
+			label leave
+			`	You turn on the external speakers. "WE ARE LEAVING," you blast to the survivors, "GET ON BOARD."`
+			`	The defeated Dragonriders need no further encouragement. They beeline for the hatch and you flicker the shield to let them on deck, where they collapse, wide-eyed and panting. Karengo isn't with them. You turn to ask about him, when a flicker of motion on a monitor catches your eye.`
+			`	Karengo is there, and he's riding a dragon, gripping the ridge of scales around its neck. The dragon is frantically trying to get him off, but deft application of his jetpack is, for now, keeping him on top of the beast. He lets out wild screams of triumph as they soar through the sky and at last you can understand the appeal of this whole expedition. Your admiration is cut short by two catastrophic crashes on either side of the ship and accompanying sirens. The dragons are trying to get through.`
+			choice
+				`	(Take off).`
+					goto takeoff
+				`	(Swear and take off).`
+				`	"Pignuts," you mutter to yourself.`
+			label takeoff
+			`	The ship takes off quickly enough to make you light-headed. "Get on board!" shouts one of the men over the external speakers as you position the ship just below Karengo and lower the shields. Scales bounce off the roof like diamonds. Karengo releases his grip on the dragon and free falls toward you, slowing at the last possible moment before dropping through the top hatch. You flick on the shields and hit the engines, hard. They respond enthusiastically, rocketing you away from the mountain and into the safety of patrolled airspace. From the cabin comes the sound of back-slapping and, surprisingly, a sob.`
+	on complete
```

Comment:
????

Reply:
Could you elaborate?

Reply:
This was meant to go on the next line, I think.